app/components/Search/Readme.md

```diff
@@ -0,0 +1,11 @@
+```
+const { Provider } = require('react-redux');
+const configureStore = require('app/utils/configureStore').default;
+const store = configureStore({ state });
```

Comment:
Guessing this isn't intended to be included in this PR?

Reply:
This was removed upstream, and should be removed. The `state-example` should also be removed from the commit it was introduced.